extensions/amp-bind/amp-bind.md

```diff
@@ -98,7 +98,7 @@ For AMP components, only select attributes are bindable:
 | Component | Attributes |
 | --- | --- |
 | amp-carousel | slide |
-| amp-img | src, srcset, alt |
+| amp-img | src, srcset, alt, height, width |
```

Comment:
Correct, only width and height for all AMP extensions for now. Looks like the implementation in `bind-validator.js` is out of date.